docs/digger/installation/macos-setup.asciidoc

```diff
@@ -1,46 +1,44 @@
-=== Setup macOS Servers
+[[macos-setup]]
```

Comment:
This file is not included in the installation guide. Should it be?

Reply:
it is https://github.com/aerogear/aerogear.org/pull/712/files

Reply:
@pmacko1 thanks, that resolves the above query

Reply:
resolved